internal/app/networkd/pkg/networkd/networkd.go

```diff
@@ -74,7 +74,11 @@ func New(config runtime.Configurator) (*Networkd, error) {
 				continue
 			}
 
-			netconf[name] = opts
```

Comment:
this allows for merging of the config options if an interface is defined multiple times.